src/com/pentalog/pentastagiu/homework2/Message.java

```diff
@@ -0,0 +1,39 @@
+package com.pentalog.pentastagiu.homework2;
+
+public class Message {
+    private String description;
+    private String author;
+    private User user;
```

Comment:
f bine ca ai adaugat relatia dintre un mesaj si un user, dar ar fi fost suficient sa ai private User author, ca sa nu ai informatii redundante(si in User sa mai ai 3 fielduri: name, username, email si sa te folosesti de obiectul User si de description atunci cand afisezi mesajul)